doc/audit/builtin_maps.rst

```diff
@@ -1,29 +1,264 @@
 Built-in Maps
 =============
 
-`public:ccf.gov.nodes.code_ids`
--------------------------------
+`public:ccf.gov.`
```

Comment:
Nitpick^2: Why italic?

Reply:
I was actually going for monospace, but I markdowned. Monospace seems good for table names? What do we think?

Reply:
Monospace is great!